examples/truffle/contracts/RustCoin.sol

```diff
@@ -1,14 +1,13 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.6.0;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
 
-contract RustCoin is ERC20, ERC20Detailed {
-  constructor() ERC20Detailed("Rust Coin", "RUST", 18) public {
+contract RustCoin is ERC20 {
+  constructor() ERC20("Rust Coin", "RUST") public {
```

Comment:
I see you noticed that Detailed was merged with ERC20. Nice catch. How did you find out? But wait, isn't this an open-zeppelin specific new thing?

Reply:
Yes, we use the openzeppeling ERC20 implementation for one of our samples.